handlebars/src/test/java/com/github/jknack/handlebars/i1084/Issue1084.java

```diff
@@ -0,0 +1,48 @@
+package com.github.jknack.handlebars.i1084;
+
+import com.github.jknack.handlebars.AbstractTest;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class Issue1084 extends AbstractTest {
```

Comment:
This is a copy of the `Issue294` test suite except uses raw expressions (triple curly brackets)